routes/Root.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+
+import { BrowserRouter, Route, Routes } from "react-router-dom";
+
+import App from "../src";
+import HomePage from "../pages/HomePage";
+import FilmPage from "../pages/FilmPage";
+import NotFoundPage from "../pages/NotFoundPage";
+
+export const Root = () => (
+  <BrowserRouter>
+    <Routes>
+      <Route path="/" element={<App />}>
```

Comment:
I think here should be the main page, for example.
**App** it's a wrapper of the app, which you can use for adding some functionality